packages/integrations/cloudflare/README.md

```diff
@@ -195,7 +195,6 @@ Determines whether and how the Cloudflare Runtime is added to `astro dev`.
 The Cloudflare Runtime includes [Cloudflare bindings](https://developers.cloudflare.com/pages/platform/functions/bindings), [environment variables](https://developers.cloudflare.com/pages/platform/functions/bindings/#environment-variables), and the [cf object](https://developers.cloudflare.com/workers/runtime-apis/request/#incomingrequestcfproperties). Read more about [accessing the Cloudflare Runtime](#access-to-the-cloudflare-runtime).
 
 - `local`: uses bindings mocking and locally static placeholdes
```

Comment:
Double checking that "placeholdes" is spelled correctly! Is this a thing, or a typo?

```suggestion
- `local`: uses bindings mocking and locally static placeholders
```